ui/src/containers/InventoryLayout.js

```diff
@@ -140,6 +171,29 @@ const InventoryLayout = (props) => {
             </TableBody>
           </Table>
         </TableContainer>
+        <InventoryFormModal
+          title='Create'
+          formName='inventoryCreate'
+
```

Comment:
Products can be passed here since we want to transfer state from the highest level component (i.e., containers).